tools/topology/topology2/platform/intel/ptl_clk_config.conf

```diff
@@ -0,0 +1,12 @@
+# TODO: Find out if this is needed!
+# With these added the alsa-utils generated blob matches with the
+# reference blob the start_control and stop_control words.
+
+Object.Base."clk_config".1 {
+	clock_warm_up	   4096
+	mclk		   0
+	warm_up_ovr	   0
+	clock_stop_delay   148
```

Comment:
not sure how relevant this 148 value is, this doesn't divide in anything useful.

Reply:
I have not seen impact from adding or removing this blob extension, also when I tried to find use for it in driver I couldn't find such. it was needed to binary match a reference blob. The stuff related to this could be in a separate commit at least for easy revert of drop from this PR. As you prefer.

Reply:
It's removed now. Having this part was a misunderstanding by me when decoding the blobs. The dma control tlv header looked like this data.